### PR TITLE
Update MeiliSearch to Meilisearch

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Hopefully the last PR on this

### CODE REVIEW

File: learn/cookbooks/aws.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ In the top-right corner, click on the **Launch instances** button to start the p
 
 You will now select which AMI or system Image to use to run your instance. Type **"meilisearch"** in the search bar and select the **Community AMIs** tab on the left sidebar.
 
-![Page titled: 'Step 1: Choose an Amazon Machine Image (AMI)'. There is only one AMI available, MeiliSearch-v0.19.0-Debian-10.3](/aws/02.select-ami.png)
+![Page titled: 'Step 1: Choose an Amazon Machine Image (AMI)'. There is only one AMI available, Meilisearch-v0.19.0-Debian-10.3](/aws/02.select-ami.png)
```
Comment (maryamsulemani97) [inline learn/cookbooks/aws.md:26]: Should we update it here? It does say `MeiliSearch` in the image

Comment (maryamsulemani97) [reply learn/cookbooks/aws.md:26]: This might go against the alt text "rules", but I guess we can make an exception here 🤷🏻‍♀️

**Review by dichotommy — APPROVED** [review]
May this be the last MeiliSearch we fix 🤞🏻

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**